easybuild/easyconfigs/m/mayavi/mayavi-4.6.2-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,68 @@
+easyblock = 'Bundle'
```

Comment:
@dquigley-warwick Are you up for updating this to use the new `PythonBundle` generic easyblock?

See https://github.com/easybuilders/easybuild-easyconfigs/pull/7134/files#diff-5a873f0b7b8cc031b3873bf2edda4ee9 for what this implies w.r.t. changes